sdk/src/main/java/com/google/cloud/dataflow/sdk/io/BigQueryIO.java

```diff
@@ -962,8 +959,8 @@ public PDone apply(PCollection<TableRow> input) {
         // Note that a presence check can fail if the table or dataset are created by earlier stages
         // of the pipeline. For these cases the withoutValidation method can be used to disable
         // the check.
-        // Unfortunately we can't validate anything early in case tableRefFunction is specified.
-        if (jsonTableRef != null && validate) {
```

Comment:
This is the same change you had me add to the Beam PR. I guess this was just a minor difference between the two code bases, from the original backport.  https://github.com/GoogleCloudPlatform/DataflowJavaSDK/commit/fcbfa83a45189bef43831f2a089882f9b9896849#diff-dd8400f63946a5a7cd725abd05b42ffeR967